### PR TITLE
Solved the challenge

### PR DESCRIPTION
**Pull request by svetlivaksevski**

### CODE REVIEW

File: js-array-methods_map/index.js [hunk 1]
```diff
@@ -24,10 +24,13 @@ const cards = [
...
-
-const questionAndAnswer = null; // [{ question: 'How often can I use <header>?', answer: 'As often as you like.'}, {...}]
-
+const lowerCaseAnswers = cards.map((element) => element.answer.toLowerCase()); // ['as often as you like.', ...]
+const questionsAndAnswersTogether = cards.map((element) => element.question + " - " + element.answer); // ["How often can I use <header>? - As often as you like.", ...]
+const questionAndAnswer = cards.map((element) => {
+return {
+question: element.question, 
+answer: element.answer
+}
```
Comment (philfreshman) [inline js-array-methods_map/index.js:33]: Fix the indents!

**Review by philfreshman — CHANGES_REQUESTED** [review]
Not approved

File: js-array-methods_map/index.js (hunk 1 above)
Comment (svetlivaksevski) [reply js-array-methods_map/index.js:33]: Did it. Thanks!

Comment (svetlivaksevski) [reply js-array-methods_map/index.js:33]: Correct. Will change in a bit!

**Pull request merged**